core/src/main/java/jenkins/model/GlobalConfigurationCategory.java

```diff
@@ -41,8 +41,12 @@
         return ExtensionList.lookup(GlobalConfigurationCategory.class);
     }
 
-    public static <T extends GlobalConfigurationCategory> T get(Class<T> type) {
-        return all().get(type);
+    public static @Nonnull <T extends GlobalConfigurationCategory> T get(Class<T> type) {
```

Comment:
Add Javadoc?
BTW, I didn't find ant usage of the method which would actually check for null.

A conservative approach would be to make the method `@CheckForNull`, but I am fine with the current impl in this particular case